src/flamegraph/color/mod.rs

```diff
@@ -339,6 +340,13 @@ pub(super) fn color(
         let reverse_name_hash = namehash(name.bytes().rev());
 
         (name_hash, reverse_name_hash, reverse_name_hash)
+    } else if deterministic {
+        use std::hash::Hasher;
+        let mut hasher = ahash::AHasher::default();
+        hasher.write(name.as_bytes());
+        let hash = hasher.finish() as f32 / u64::MAX as f32;
+
+        (hash, hash, hash)
```

Comment:
One consequence of setting these three to the exact same value is that the different RGB components are all scaled the same way. You'd get more variance if you generated a second (deterministic) hash and used that as `v2` and `v3`. One option would be to do:
```rust
hasher.write_u8(0);
let hash2 = hasher.finish() ...

(hash, hash2, hash2)
```

Reply:
This fixes the color selection oddness.